spec/factories.rb

```diff
@@ -1,13 +1,13 @@
 FactoryGirl.define do
-  factory :actor do
+  factory :new_actor, class: Actor do
   end
 
-  factory :character do
+  factory :new_character, class: Character do
   end
 
-  factory :director do
+  factory :new_director, class: Director do
   end
 
-  factory :movie do
+  factory :new_movie, class: Movie do
```

Comment:
It shouldn't be necessary to rename these, as long as they are not in `spec/factories/[MODEL NAME].rb`. Right? Or did you observe them being ignored?